cli.js

```diff
@@ -12,9 +12,11 @@ const cli = meow(`
 	  $ echo <string> | chalk <style> …
 
 	Options
-	  --template, -t  Style template. The \`~\` character negates the style.
+		--template, -t  Style template. The \`~\` character negates the style.
+		--demo, -d Demo of chalk possibilities.
```

Comment:
Needs a better description. And no point in having a short flag for this.

Reply:
The help output needs to be updated in the readme too.

Reply:
What do you want as a description then, would be easier than trying ti figure it out :)

Reply:
> Demo of all Chalk styles